src/fsdocs-tool/BuildCommand.fs

```diff
@@ -1403,26 +1403,10 @@ type CoreBuildOptions(watch) =
 
         let userParametersDict = readOnlyDict userParameters
 
-        // Adjust the user substitutions for 'watch' mode root
-        let userRoot, userParameters =
-            if watch then
-                let userRoot = sprintf "http://localhost:%d/" this.port_option
-
-                if userParametersDict.ContainsKey(ParamKeys.root) then
-                    printfn "ignoring user-specified root since in watch mode, root = %s" userRoot
-
-                let userParameters =
-                    [ ParamKeys.root, userRoot ]
-                    @ (userParameters |> List.filter (fun (a, _) -> a <> ParamKeys.root))
-
-                Some userRoot, userParameters
-            else
-                let r =
-                    match userParametersDict.TryGetValue(ParamKeys.root) with
-                    | true, v -> Some v
-                    | _ -> None
-
-                r, userParameters
+        let userRoot =
```

Comment:
This will clash later on with

https://github.com/fsprojects/FSharp.Formatting/blob/2efb355823b8311f91a06566c0a3de86037a590d/src/fsdocs-tool/ProjectCracker.fs#L611-L614

Just following the instructions in the readme (`dotnet build` and `src\fsdocs-tool\bin\Debug\net6.0\fsdocs.exe watch`) will launch everything pointing to `https://fsprojects.github.io/FSharp.Formatting` and not localhost.